app/DynamicsAdapter/DynamicsAdapter.Web/SearchRequest/SearchRequestJob.cs

```diff
@@ -32,15 +35,17 @@ public async Task Execute(IJobExecutionContext context)
             _logger.LogInformation("Search Request started!");
 
 
-            _logger.LogDebug("Attempting to get token");
-            var token = await _oAuthApiClient.GetRefreshToken(CancellationToken.None);
-            _logger.LogInformation($"Successfully got token from provider, token expires in {token.ExpiresIn}");
+            _logger.LogDebug("Attempting to get search request from dynamics");
+            var request = await _dynamicsApiClient.Get<SSG_SearchRequests>(
+                $"ssg_searchrequests({Guid.NewGuid()})");
+            _logger.LogInformation($"Successfully got entity from dynamics");
 
             _logger.LogDebug("Attempting to post person search");
             var result = await _searchApiClient.SearchAsync(new PersonSearchRequest()
```

Comment:
add if (request !=null) to avoid NullException

Reply:
I'm not using request anywhere now, so don;t need to check for null ;-)

Reply:
request.SSG_PersonGivenName,

Reply:
FirstName = request.SSG_PersonGivenName,
                LastName = request.SSG_PersonSurname,
                DateOfBirth = request.SSG_PersonBirthDate
those are not used anymore?